src/EventService/appsettings.json

```diff
@@ -0,0 +1,56 @@
+{
+  "Serilog": {
+    "Using": [],
+    "LevelSwitches": { "$controlSwitch": "Information" },
+    "MinimumLevel": { "ControlledBy": "$controlSwitch" },
+    "Enrich": [
+      "FromLogContext",
+      "WithMachineName",
+      "WithProcessId",
+      "WithThreadId"
+    ],
+    "WriteTo": [
+      {
+        "Name": "Console"
+      },
+      {
+        "Name": "Seq",
+        "Args": {
+          "serverUrl": "http://localhost:5341",
+          "apiKey": "PRLXBdmLmIGBbDsD3AkT"
+        }
+      }
+    ]
+  },
+  "ConnectionStrings": {
+    "Redis": "localhost:6379"
+  },
```

Comment:
don't need redis for now, remove lines 25-27

Reply:
Removed